starknet_devnet/blueprints/rpc/rpc_spec.py

```diff
@@ -7,7 +7,7 @@
     "openrpc": "1.0.0-rc1",
     "info": {
         "version": "0.45.0",
-        "title": "StarkNet Node API",
+        "title": "Starknet Node API",
```

Comment:
@cptartur I feel like we haven't kept the specs copy-pasted so far, so this change could be acceptable? Or do we only want to update the specs when a new spec version is released?

Reply:
I think it's okay to change it 👍